src/controllers/credentials.ts

```diff
@@ -423,7 +472,35 @@ export class CredentialController {
 		}
 
 		const { presentation, verifierDid, policies } = request.body;
-		const verifyStatus = request.query.verifyStatus === 'true' ? true : false;
+		const verifyStatus = request.query.verifyStatus === 'true';
+		const allowDeactivatedDid = request.query.allowDeactivatedDid === "true";
+
+		let issuerDid = "";
+		let decoded: any;
+
+		if (presentation.issuer?.id) {
+			issuerDid = presentation.issuer.id;
+		} else {
+			try {
+				decoded = jwt_decode(presentation);
```

Comment:
The same comment, let's move it to the helper function

Reply:
Moved to the helper.ts. ✅